scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -498,6 +521,13 @@ object Execution {
   private case class WriteExecution[T](head: ToWrite, tail: List[ToWrite], fn: (Config, Mode) => T) extends Execution[T] {
 
     /**
+     * override this so that we can push down the conf value to the current TypedPipes
+     */
+    override def setExtraConfig(conf: Config): Unit = {
+      (head :: tail).foreach(_.setExtraConfig(conf))
```

Comment:
But this doesn't push it down to that, it attempts to push it onto the final rich pipe thats doing the write out, which won't effect any intermediate mappers , reducers or other things in this execution.

I'm not really sure this is workable, I don't think any of the API's in cascading allow for accurate targeting of steps correctly. Then there is also what happens if you zip two executions together with competing opposite options? 

@johnynek  thoughts?